.github/workflows/pipeline.yml

```
@@ -1,136 +1,220 @@
-name: Build and lint
-
+name: Build, lint and typecheck examples
+ 
 on:
     # Triggers the workflow on push or pull request events but only for the main branch
     push:
         branches: [main, release**]
     pull_request:
         branches: [main, release**]
-
+        # Don't run on draft PR's, see: https://github.com/orgs/community/discussions/25722#discussioncomment-3248917
+        types: [opened, synchronize, reopened, ready_for_review]
     # Allows us to run the workflow manually from the Actions tab
     workflow_dispatch:
 
 env:
+  NODE_VERSION: 18.16.0
+  RUST_VERSION: 1.62
   RUST_FMT: nightly-2022-06-09-x86_64-unknown-linux-gnu
```

We should update this to match the formatter we changed the rest of our projects to run with:
```suggestion
  RUST_FMT: nightly-2023-04-01-x86_64-unknown-linux-gnu
```

---

I agree but then we need to bump concordium-base as well, which is not really a problem, I'll just do that.

---

I think we should :)